### PR TITLE
yandex map was added

### PR DESCRIPTION
**Pull request by Adroneleh**
I added a map to the main page

### CODE REVIEW

File: event-app/src/App.vue [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <template>
   <div id="app">
-    <img alt="Vue logo" src="./assets/logo.png">
-
+   <!-- <img alt="Vue logo" src="./assets/logo.png"> -->
```
Comment (Zimtir) [inline event-app/src/App.vue]: remove a **commented** code

File: event-app/src/App.vue [hunk 2]
```diff
@@ -1,7 +1,7 @@
...
+   <map-component></map-component>
```
Comment (Zimtir) [inline event-app/src/App.vue]: nav-component must be **upper** than map-component

File: event-app/src/App.vue [hunk 3]
```diff
@@ -16,14 +16,14 @@
...
-
 import NavComponent from "./components/main/nav/NavComponent.vue";
+import MapComponent from "./components/main/map/MapComponent.vue";
 
 export default {
   name: "app",
   components: {
     NavComponent,
-
+    MapComponent,
```
Comment (Zimtir) [inline event-app/src/App.vue]: It's our component and need in a placing in our component-block instead injection-component-block

File: event-app/src/components/main/map/MapComponent.vue [hunk 4]
```diff
@@ -0,0 +1,71 @@
+<template>
+<div class="map-container">
+ <yandex-map 
+  :coords="[54.62896654088406, 39.731893822753904]"
```
Comment (Zimtir) [inline event-app/src/components/main/map/MapComponent.vue]: Create markers as array and use v-for loop in the future.
Now it's not required

Comment (Adroneleh) [reply event-app/src/components/main/map/MapComponent.vue]: Also was added mock data (marker examples)

**Pull request merged**